src/components/LoginButton/LoginButton.stories.mdx

```diff
@@ -22,6 +22,12 @@ import LoginButtonIcon from './LoginButtonIcon';
         type: 'text',
       },
     },
+    appearance: {
+      options: LOGO_APPEARANCES,
+      control: {
+        type: 'radio',
+      },
+    },
```

Comment:
@jiromaykin I exported the possible values for `appearance` so we can feed that constraint to Storybook - now you can just select the appearance between two radio inputs.